### PR TITLE
Feature/cal 016

### PR DESCRIPTION
**Pull request by tomascravero**

### CODE REVIEW

File: server/controllers/buildings.js [hunk 1]
```diff
@@ -13,13 +13,24 @@ exports.findAll = (req, res) => {
...
+  if (!req.body.address || !req.body.fullname || !req.body.phoneNumber) {
     res.status(400).send({ message: "Content can't be empty " });
   }
+  if(!req.body.adress.match(addressValid) || req.body.adress.length < 3) {
+    return res.status(400).send({ message: "Error: Invalid Adress"})
+  }
+  if(req.body.phoneNumber.length < 7) {
+    return res.status(400).send({ meessage: "Error: phone not valid"})
+  }
+  if(req.body.fulllname.length < 3) {
```
Comment (Alfondalix) [inline server/controllers/buildings.js:26]: fullName

File: server/controllers/buildings.js [hunk 2]
```diff
@@ -13,13 +13,24 @@ exports.findAll = (req, res) => {
...
+    return res.status(400).send({ message: "Error: Invalid Builgind Name"})
+  }
+
   const building = new Building({
     address: req.body.address,
     boilers: req.body.boilers,
-    fullName: req.body.fullName,
+    fullname: req.body.fullname,
```
Comment (Alfondalix) [inline server/controllers/buildings.js]: fullName

File: server/controllers/buildings.js [hunk 3]
```diff
@@ -53,14 +64,19 @@ exports.findOne = (req, res) => {
...
-    return res.status(400).send({
-      message: "Data can't be empty",
-    });
-  }
-  if (!req.body.address || !req.body.fullName || !req.body.phoneNumber) {
+  const addressValid = /\w\s\d/;
+  if (!req.body.address && !req.body.fullname && !req.body.phoneNumber) {
     return res.status(400).send({ message: "content can't be empty" });
   }
+  if (req.body.fulllname.length < 3) {
```
Comment (Alfondalix) [inline server/controllers/buildings.js:71]: fullName

**Review by Alfondalix — APPROVED** [review]

**Review by Ignacio-Sormani — COMMENTED** [review]
-I think some error messages are with the first letter of each word with uppercase and other error messages have all the letters in lowercase. Make them consistent.
-We should add model validations such as: unique, match or default.
-We should also update postman collection with the changes that are made in this branch (fullName and cuit in body of requests)

**Review by Ignacio-Sormani — APPROVED** [review]

**Pull request merged**